packages/interbit-ui-components/src/tests/reduxForm.test.js

```diff
@@ -0,0 +1,72 @@
+import assert from 'assert'
+import validation from '../components/ReduxForm/validation'
+
+describe('ReduxForm component', () => {
+  describe('IbField component', () => {
+    // TODO #694: implement the following tests
+    it('renders an <input type="text" /> when type="text"', () => {})
+    it('renders a <select /> and <option />s when type="select"', () => {})
+    it('renders a <select multiple /> and <option />s when type="select" and multiple={true}', () => {})
+    it('renders an <input type="checkbox" /> when type="checkbox"', () => {})
+    it('renders an <input type="radio" /> when type="radio"', () => {})
+  })
+
+  describe('Validation functions', () => {
+    it('checks for valid email formats in validation.email', () => {
+      const validEmails = ['meow@meow.com', '', undefined, null]
```

Comment:
I really don't understand how '', undefined and null can be valid email addresses. Perhaps `validEmptyValues`.